DICOM/src/Microsoft.Health.Anonymizer.Common/Models/DateTimeGlobalSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Anonymizer.Common.Models
+{
+    public static class DateTimeGlobalSettings
```

Comment:
If this is a type of setting, can we move it in the folder of settings instead of models?

Reply:
Or maybe we can rename it to `DateTimeConstants` to avoid similar concepts with classes in settings. Besides, we can make these properties readonly, as they are assumed to be fixed values that are not configured by users.

Reply:
Moved it into settings, but it could be set by users not constants.